third_party/cudnn/local/BUILD

```diff
@@ -19,6 +26,7 @@ cc_import(
     name = "cudnn_lib",
     shared_library = select({
         ":aarch64_linux": "lib/aarch64-linux-gnu/libcudnn.so",
+        ":windows": glob(["bin/cudnn64_*.dll"])[0],
```

Comment:
Just to note, the glob failed, I ended up just putting the 7 in the path

Reply:
How do we support cudnn8?
I was trying to make sure it supports v8 and the glob worked for me to pick 7 or 8.

I'm not good with bazel and couldn't figure out how to do a wildcard without a glob. Then a glob returns a list so I needed to get the first value.

Reply:
> Also just curious when you do development on Windows do you typically install libraries like cudnn or tensorrt to the system somewhere and reuse them or do you keep them in a workspace that is project specific. Seems like we could extend these changes to the archive based dependencies too pretty easily. Doesn't have to be in this PR but just wondering if it would be useful.

I installed cudnn and tensorrt to a folder and specified it in WORKSPACE. I'm more of a Linux developer but need this working on Windows if we are to use it at all.

Not sure what's going on with the stream operator there. From the error, it looks like it doesn't like:
C++```
#define TRTORCH_LOG(l, sev, msg)               \
    do {                                       \
        std::stringstream ss{};                \
        ss << msg;                             \
        l.log(sev, ss.str());                  \
    } while (0)
```

Maybe a MSVC issue (if that's the compiler you are using).
If you remove the line:
`ss << msg;`
does it compile?

Reply:
I can look into if there is another way to get the wildcard working. Hmm, I can look into the logging issue more, you didn't hit it right? What version of MSVC did you use? Did you add any additional flags or anything (like I noticed MSVC doesnt handle default flags we have like `--std=c++14`)?

Reply:
@narendasan Any update with this?
It seems to work fine here. Is it the version of Bazel I am using?

Reply:
Seems like something about the particular use of a specialized logger in conversion.cpp that is causing the issue, If I switch to the generic logger I can compile the library and successfully start programs using the core library such as `trtorchc` (I don't notice the issue mentioned in #153 if i understood the issue correctly either, the compiler starts up correctly but I think we don't handle windows paths well so i haven't yet tested further). I dont think it is due to the bazel version as it just calls the underlying C++ compiler similar to how make or CMake does. I think its a compiler issue, I am not sure which compiler version you used, I used the latest MSVC 2019 build tools. If there is anymore information you could give me about your build environment and build process that would be helpful. I think other than that issue this PR is good to merge though.

Reply:
I mean for the globbing issue. Is this related to the bazel version?

Reply:
No I think it may have been a path error on my part i think. It works now.